edxval/tests/test_api.py

```diff
@@ -269,6 +275,94 @@ def test_get_url_for_profile(self):
         self.assertEqual(url, u'http://www.meowmix.com')
 
 
+class GetVideoForCourseProfiles(TestCase):
+    """Tests get_video_info_for_course_and_profiles in api.py"""
+
+    def setUp(self):
+        """
+        Creates a course for testing
+
+        Creates two videos, each with two encodings.
+        """
+        Profile.objects.create(**constants.PROFILE_DICT_MOBILE)
+        Profile.objects.create(**constants.PROFILE_DICT_DESKTOP)
+        # 1st video
+        video = Video.objects.create(**constants.VIDEO_DICT_FISH)
+        EncodedVideo.objects.create(
+            video=Video.objects.get(
+                edx_video_id=constants.VIDEO_DICT_FISH.get("edx_video_id")
+            ),
+            profile=Profile.objects.get(profile_name="mobile"),
```

Comment:
Save the result of `create` above to a variable and use it here instead of doing a get